src/common/personalizedDigest.ts

```diff
@@ -126,6 +129,8 @@ const getEmailVariation = async ({
       username: user.username,
       image: user.image,
       reputation: user.reputation,
+      currentStreak: userStreak?.currentStreak,
+      maxStreak: userStreak?.maxStreak,
```

Comment:
nvm, just realised this is a snapshot test

Reply:
They can be missing in the db right? So maybe fallback to `0`.

Reply:
We want to have null values here because it's the easiest way to determine whether streaks should be shown in the digest or not (if the feature is enabled or not for the user)

Reply:
@DragosIorgulescu did you get that request from product or?

Reply:
fair point, I'll add a separate attribute dictating that 👍